apps/bottle_app.py

```diff
@@ -0,0 +1,24 @@
+from bottle import Bottle, run, redirect
+from vulnpy.bottle.vulnerable_routes import add_vulnerable_routes
+
+
+app = Bottle(__name__)
+add_vulnerable_routes(app)
+
+
+@app.route("/")
+def index():
+    redirect("/vulnpy")
+
+
+class StripPathMiddleware(object):
+    def __init__(self, app):
+        self.app = app
+
+    def __call__(self, e, h):
+        e["PATH_INFO"] = e["PATH_INFO"].rstrip("/")
+        return self.app(e, h)
+
+
+if __name__ == "__main__":
+    run(app=StripPathMiddleware(app))
```

Comment:
eventually we'll need to use contrast's bottle middleware here too - we'll be forced to do so by the integration (app) test pipeline. For now though there's nothing to do yet!